.github/workflows/integration-tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         ports:
           - 16379:6379
       redis-cluster:
-        image: grokzen/redis-cluster:latest
+        image: grokzen/redis-cluster:6.0.0
```

Comment:
I guess using `latest` is not always the best way, because you don't know with which version the tests were running. In this image the `latest` tag should be `7.0.7` for now, but it doesn't match that version. And we are using redis 6.0.0 for all of our redis tests, so we should do it here this way as well.

Reply:
We should probably use 6.2.8 instead of 6.0.0. According to https://relay.so/docs/1.x/installation#requirements, Relay requires Redis 6.2.7+

Reply:
Then we should update redis and redis-cluster to this version IMHO. Maybe this fixes the skipped relay tests, I'll try this tomorrow.

Reply:
I updated to 6.2.8. But the skipped tests are still failing.